tests/endToEnd.py

```diff
@@ -212,13 +218,15 @@ def testEndToEnd(self):
         #        datarel needs imsim_*.
         ver = 'imsim-2010-12-17-1'
         print "Setting up astrometry_net_data", ver
+        # XXX what is actually used from this setup -- a path in the env?
```

Comment:
`meas_astrom` and `datarel` both seem to be checking for `ASTROMETRY_NET_DATA_DIR` environment variable. I guess the `setup` is technically meant to set up all the dependencies of `astrometry_net_data` but the eups tables file doesn't list any of note. It seems that this could just do a `getPackageDir("astrometry_net_data")` and wait for it to throw if that environment variable is missing.